vcx/dummy-cloud-agent/src/actors/requester.rs

```diff
@@ -10,6 +10,10 @@ use actors::RemoteMsg;
 use futures::*;
 use utils::futures::*;
 
+lazy_static! {
+    pub static ref REQWEST_CLIENT : reqwest::r#async::Client = reqwest::r#async::Client::new();
```

Comment:
may be we can just update reqwest to newer version and avoid escaping `async` reserved word?